README.md

```diff
@@ -3,26 +3,27 @@
 ## Install
 
 ```bash
-git clone ...
+git clone https://github.com/TomasVotruba/phpfwtrends.org
 composer install
 ```
 
-## Generate Stats
+## Generate stats
 
 ```bash
 bin/console generate-stats
 ```
 
-## Run Website Locally
+## Run website locally
 
 ```bash
-php -S localhost:8001 -t public 
+cd path/to/clone/directory
+php -S 0.0.0.0:8001 -t public public/index.php
 ```
 
-Open [localhost:8001](http://localhost:8001) in your browser to see the website
+Open [0.0.0.0:8001](http://0.0.0.0:8001) in your browser to see the website
 
-## Create Static Website to Deploy
+## Create static website to deploy
```

Comment:
English Headlines must be capical

Reply:
@TomasVotruba  you are right but I found my self for the english  sentence formation like **The First Letter in a Sentence should be captial**

Reply:
There are different styles and the current one looks like the Chicago Manual of Style.

Here is a tool that helps with capitalization in titles and which describes the different styles: https://capitalizemytitle.com

Reply:
Lets add style code into contribution guide line.